api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -103,8 +104,15 @@ public void syncBillingProjectStatus() {
           default:
             break;
         }
+      } catch (NotFoundException e) {
```

Comment:
opt: One thing that I found would have been useful when debugging this issue is if L140 had some associated logging. One theory I had is we were hitting the Creating timeout.